elan/rpc/rpc.go

```diff
@@ -204,6 +204,18 @@ func (s *server) GetCapabilities(ctx context.Context, req *pb.GetCapabilitiesReq
 			ActionCacheUpdateCapabilities: &pb.ActionCacheUpdateCapabilities{
 				UpdateEnabled: false,
 			},
+			CachePriorityCapabilities: &pb.PriorityCapabilities{
+				Priorities: []*pb.PriorityCapabilities_PriorityRange{
+					{
+						MinPriority: 0,
+						MaxPriority: 0,
+					},
+					{
+						MinPriority: -1,
+						MaxPriority: -1,
+					},
+				},
+			},
```

Comment:
Ah I did see this but wondered what it was about!

Reply:
Yeah it's a bit arbitrary and we don't actually use this response, but I think it makes sense for us to advertise it given we're now attaching some semantic meaning to it.